src/datastructure.h

```diff
@@ -15,8 +15,9 @@
 
 // enum privacy_level
 #include "enums.h"
-// assert_sizeof
-#include "static_assert.h"
+
+// Definitions like OVERTIME_SLOT
+#include "FTL.h"
```

Comment:
Why is this needed now?

Reply:
See https://github.com/pi-hole/FTL/pull/1329#discussion_r857076488